tools/google-cloud-support-slackbot/google_cloud_support_slackbot.py

```diff
@@ -0,0 +1,811 @@
+#!/usr/bin/env python3
```

Comment:
This file is huge. Is there any way to split it in different files and still work as a Slack bot?

Reply:
It should be doable. The APIs I"m calling are currently in alpha and the Support team working on them has informed me that they may change when they go to GA, so I was planning to modularize everything at that point for the v1 of the bot